librocksdb_sys/crocksdb/c.cc

```diff
@@ -2595,6 +2625,30 @@ void crocksdb_options_set_vector_memtable_factory(crocksdb_options_t* opt, uint6
   opt->rep.memtable_factory.reset(new VectorRepFactory(reserved_bytes));
 }
 
+void crocksdb_load_latest_options(const char* dbpath,
```

Comment:
We need to distinguish three cases:
1. the options file is loaded successfully
2. the options file (or the dbpath) is not found
3. there is some error related to the options file

Reply:
Of course, the errptr will shows the exactly err msg

Reply:
I think it's not a good idea to rely on the error message, you can return a bool explicitly.